README.md

```diff
@@ -16,4 +16,7 @@ Earth Enterprise Fusion & Server currently run on 64-bit versions of the followi
 Refer to the [wiki](https://github.com/google/earthenterprise/wiki/Build-Instructions) for instructions on building from source on one of these platforms.
 
 ### Installation
-There is currently no official installer for Earth Enterprise Fusion & Server.  For details and workarounds, refer to the [FAQ](https://github.com/google/earthenterprise/wiki/Frequently-Asked-Questions#ive-built-gee-fusion-and-server-now-how-do-i-install-them).
\ No newline at end of file
+There is currently no official installer for Earth Enterprise Fusion & Server.  For details and workarounds, refer to the [FAQ](https://github.com/google/earthenterprise/wiki/Frequently-Asked-Questions#ive-built-gee-fusion-and-server-now-how-do-i-install-them).
+
+### Known Issues
+Known issues can be tracked via the GitHub Issues page or the weekly updated [wiki page](https://github.com/google/earthenterprise/wiki/Known-Issues).
```

Comment:
Nit: the wording here is a bit awkward ("the weekly updated wiki page").  It might be better to say something like "the wiki page, which is updated weekly".

There are also some odd redundancies in the linked wiki page ("tracking page for issues being tracked", "packaged with the release when a release is packaged").  It would be nice if this could be rewritten to use the words track, release, and package only once.